### PR TITLE
docs: Update CLI options

### PR DESCRIPTION
**Pull request by kachick**
Follow changes in #31 and #85

### CODE REVIEW

**Comment by andrewbranch** [comment]
Thank you! Could you add a changeset per the comment above, since the README of the published package is what shows on npmjs.com?

File: packages/cli/README.md [hunk 1]
```diff
@@ -147,7 +147,6 @@ Specifies rules/problems to ignore (i.e. not raise an error for).
 
 The available values are:
 
-- `wildcard`
```
Comment (kachick) [inline packages/cli/README.md:150]: I can't find the diff in #31 and #85 for this option.

**Comment by kachick** [comment]
Added the changeset and squashed commit again!

This is my first experience for using changeset. I didn't know the toolset.
Could you check again? 🙏

**Review by andrewbranch — APPROVED** [review]

**Comment by kachick** [comment]
Thank you!

And thank you for this project. This helped me a lot in understanding how to test package behavior in octokit/graphql-schema#925 🙏
Initially, I was fighting with `npm link`.

**Pull request merged**